packages/api/src/functions/dbAuth/DbAuthHandler.ts

```diff
@@ -17,7 +17,7 @@ interface DbAuthHandlerOptions {
    * The name of the property you'd call on `db` to access your user table.
    * ie. if your Prisma model is named `User` this value would be `user`, as in `db.user`
    */
-  authModelAccessor: string
+  authModelAccessor: keyof PrismaClient
```

Comment:
I'm no Typescript expert, BUT: is it a key of `PrismaClient`, or a key of an *instance* of `PrismaClient`? The `authModelAccessor` is a key for our standard `db` var that's available everywhere, but `db` is created as:

```javascript
export const db = new PrismaClient({
  log: emitLogLevels(['info', 'warn', 'error']),
})
```

Reply:
Or maybe there's no difference as far as Typescript is concerned? <shrug>

Reply:
I'm thinking it works something like this:

```ts
class PrismaClient {
  constructor() {}
  users = ['alice', 'bob']
  posts = ['post one', 'post two']
}

const db = new PrismaClient()

let key: keyof PrismaClient

key = "users"

console.log(db[key])
```
https://www.typescriptlang.org/play?#code/MYGwhgzhAEAKBOBLCBbMBhEiCmA7ALtAN4BQ00wA9rhPvAK7D6XwAUAlMQL5nT0TZ4MALzQA2gHIwWYNgkAaaBIBGlZRIC6vAA6VaI8RN21o1OYqN7C+AO6VNJHiSo1CAE2XRRubDbhJUDCw8fA4SEhBsQgBrbABPAC5oWLjKADN-ZDRMHAJwlK9oACJ+QQgi8JcISkiAOhBKAHNWDzEUjXYSIA

Reply:
Sounds good to me!